cmake/std/atdm/contributed/kokkos-dev-2/environment.sh

```diff
@@ -196,7 +196,7 @@ elif [[ "$ATDM_CONFIG_COMPILER" == "INTEL-18.0.5" ]] ; then
   export ATDM_CONFIG_BLAS_LIBS="-mkl"
 elif [[ "$ATDM_CONFIG_COMPILER" == "INTEL-19.0.5" ]] ; then
   module load sems-archive-gcc/7.2.0
-  module load sems-archive-intel/18.0.5
+  module load sems-archive-intel/19.0.5
   module load atdm-env
   module load atdm-mkl/18.0.5
```

Comment:
Should the mkl version also be changed?

Reply:
No, SEMS doesn't have a corresponding mkl module for the later version.